### DISCUSSION TITLE
Distro packages

### DISCUSSION THREAD

**Discussion by phsm** [OvenMediaLabs/OvenMediaEngine#1382] [Ideas] [answered]
Hello, 

Have you considered building .deb and .rpm packages / making repositories?

I first tried making a .deb package myself, but then I discovered the dependencies are built from sources instead of using distro-provided versions which makes building proper packages hard. Is there a strict requirement for specific library versions?

**Comment by getroot** [comment, marked as answer]
We do not yet have plans to release .deb or .rpm packages for the open source version. The reason, as always, is a matter of operating costs. If someone could contribute it, many people, including me, would welcome it.

There are no strict requirements for the versions of dependent packages. We just haven't tested other versions. However, it will probably only work if you match the major versions of OpenSSL (3) and FFmpeg (5).

For your information, archlinux has a wonderful contributor who directly maintains the OvenMediaEngine package. Please note the following:
https://archlinux.org/packages/extra/x86_64/ovenmediaengine/
